test/e2e/tests/send-eth.spec.js

```diff
@@ -129,7 +129,7 @@ describe('Send ETH non-contract address with data that matches ERC20 transfer da
 
```

Comment:
the test changes restore the tests to the state they were in before the previous PR linked in the above description of this PR